serenata_toolbox/federal_senate/federal_senate_dataset.py

```diff
@@ -0,0 +1,100 @@
+import os.path
+from urllib.request import urlretrieve
+import numpy as np
+import pandas as pd
+
+from datetime import date
+
+class FederalSenateDataset:
+    URL = 'http://www.senado.gov.br/transparencia/LAI/verba/{}.csv'
+    FIRST_YEAR = 2008
+    NEXT_YEAR = date.today().year + 1
+
+    YEAR_RANGE = range(self.FIRST_YEAR, self.NEXT_YEAR)
+
+    def __init__(self, path):
+        self.path = path
+
+    def fetch(self):
+        for year in YEAR_RANGE:
+            url = self.URL.format(year)
+            file_path = os.path.join(self.path, 'federal-senate-{}.csv'.format(year))
+            urlretrieve(url, file_path)
+
+    def translate(self):
+        filenames = ['federal-senate-{}.csv'.format(year) for year in YEAR_RANGE]
+        for filename in filenames:
+            csv_path = os.path.join(self.path, filename)
+            self.__translate_file(csv_path)
+
+    def clean(self):
+        reimbursement_path = os.path.join(self.path, 'federal-senate-reimbursements.xz')
+
+        filenames = ['federal-senate-{}.xz'.format(year) for year in YEAR_RANGE]
+        dataset = pd.DataFrame()
+
+        for filename in filenames:
+            file_path = os.path.join(self.path, filename)
+            data = pd.read_csv(file_path, encoding = "utf-8")
+            dataset = pd.concat([dataset, data])
+
+        dataset['date'] = pd.to_datetime(dataset['date'], errors='coerce')
+        dataset['cnpj_cpf'] = dataset['cnpj_cpf'].str.replace(r'\D', '')
+
+        dataset.to_csv(reimbursement_path, compression='xz', index=False, encoding='utf-8')
+
+        return reimbursement_path
+
+    def __translate_file(self, csv_path):
```

Comment:
This internal method looks a little big, could we break it to make the parts smaller and more meaningful and clear?